packages/itwin/one-click-lca-widget/src/components/ExportModal.tsx

```diff
@@ -129,13 +129,13 @@ const ExportModal = (props: ExportProps) => {
           }
         } catch (e) {
           setJobStatus(JobStatus.StatusEnum.Failed);
-          toaster.negative("You are not authorized to create carbon calculation jobs. Please contact project administrator.");
+          toaster.negative("You do not have required permissions. Please contact project administrator.");
```

Comment:
You do not have **the** required permissions. Please contact **the** project administrator.